### ISSUE TITLE
Not all URLs are turned into links in the chat

### ISSUE THREAD

**Issue by ancorgs** [jangouts/jangouts#412]
We are using Interweave that, by default, only turn into links URLS [from a small set of TLDs](https://github.com/milesj/interweave/blob/master/website/docs/exts/autolink.mdx#tld-support). It feels like too much. We need to either:

- Expand that list of TLDs
- Disable TLD validation

**Comment by dgdavid** [comment]
We need to either:
> 
>     * Expand that list of TLDs
> 
>     * Disable TLD validation

We could make it configurable too through the `config.json` file, allowing either

```json
{
  "joinUnmutedLimit": 3,
  "enableTLDValidation": true,
  "expandTLDValidation": ["app", "bio", "online"],
}
```


For our instance, I vote for disabling the TLD validation.

**Comment by dgdavid** [comment]
> We could make it configurable too through the `config.json` file

Looking for a better name than above that helps to clarify that is the TLD validation for the links shared in the chat.

**Comment by ancorgs** [comment]
> We could make it configurable too through the `config.json` file, allowing either

I'm not sure that's a good idea. It tights us to interweave. If we want to use a different library in the future we have to restrict ourselves to a library that also contains TLD validation or implement something compatible ourselves.

> For our instance, I vote for disabling the TLD validation.

I would just disable it in Jangouts in general for the time being. There is always time to introduce it in a future version if we detect there is a real need for TLD validation.

**Comment by dgdavid** [comment]
> > We could make it configurable too through the `config.json` file, allowing either
> 
> I'm not sure that's a good idea. It tights us to interweave. If we want to use a different library in the future we have to restrict ourselves to a library that also contains TLD validation or implement something compatible ourselves.
> 

Right, good point.

> > For our instance, I vote for disabling the TLD validation.
> 
> I would just disable it in Jangouts in general for the time being. There is always time to introduce it in a future version if we detect there is a real need for TLD validation.

+1

**Comment by ancorgs** [comment]
Fixed (or at least mitigated) by #416
